app/models.py

```diff
@@ -642,6 +644,7 @@ class User(db.Model):
     password_changed_at = db.Column(db.DateTime, index=False, unique=False,
                                     nullable=False)
     logged_in_at = db.Column(db.DateTime, nullable=True)
+    terms_accepted_at = db.Column(db.DateTime, index=False, nullable=False, default=datetime.utcnow)
```

Comment:
related to other PR, this could be terms_accepted_version as int

we just want to know which version they accepted